### PR TITLE
Review initial posts and profiles based on writing guide

### PR DESCRIPTION
**Pull request by 454de6e**
Issue: https://github.com/UNDataForum/main-website/issues/31

### CODE REVIEW

**Comment by 454de6e** [comment]
Posts:
- https://main-website-git-issue-31.undataforum.now.sh/blog/reaching-for-new-heights-in-dubai/
- https://main-website-git-issue-31.undataforum.now.sh/blog/mobile-phone-network-data-for-development/
- https://main-website-git-issue-31.undataforum.now.sh/blog/the-power-of-new-data-sources/
- https://main-website-git-issue-31.undataforum.now.sh/blog/inclusive-data-charter/
- https://main-website-git-issue-31.undataforum.now.sh/blog/public-private-partnerships-in-statistical-capacity-development/
- https://main-website-git-issue-31.undataforum.now.sh/blog/big-data-for-sustainable-development/
- https://main-website-git-issue-31.undataforum.now.sh/blog/weighing-the-quality-of-big-data/

Profiles:
- https://main-website-git-issue-31.undataforum.now.sh/profiles/jennie-bell/
- https://main-website-git-issue-31.undataforum.now.sh/profiles/pietro-bertazzi/
- https://main-website-git-issue-31.undataforum.now.sh/profiles/paula-hidalgo-sanchis/
- https://main-website-git-issue-31.undataforum.now.sh/profiles/robert-kirkpatrick/
- https://main-website-git-issue-31.undataforum.now.sh/profiles/francesca-perucci/
- https://main-website-git-issue-31.undataforum.now.sh/profiles/syed-raza/
- https://main-website-git-issue-31.undataforum.now.sh/profiles/matthew-shearing/
- https://main-website-git-issue-31.undataforum.now.sh/profiles/shyam-upadhyaya/

File: content/posts/public-private-partnerships-in-statistical-capacity-development.md [hunk 1]
```diff
@@ -29,14 +29,14 @@ There are four possible areas of trust-building we need to examine. Like most su
...
+In Dubai, I will be speaking on this topic in the session [Impact measurement, SDGs and support to official statistics – the role of private sector and partnerships (TA1.04)](/2018/sessions/ta1.04/) run by the [GOPA Consulting Group](https://www.gopa-group.org/), a technical service provider in capacity-building, jointly with [UNDP](https://www.undp.org/). This session also includes speakers from an NSO and representatives from other sectors. This session and other sessions will provide a valuable opportunity for us all to share evidence, ideas and perspectives. I also look forward to receiving responses to this blog post with your thoughts and ideas to keep the conversation going.
```
Comment (gonzalezmorales) [inline content/posts/public-private-partnerships-in-statistical-capacity-development.md]: How are we supposed to deal with the links to pages of the undataforum website that are not yet in place? (e.g., /2018/sessions/ta1.04/)?  Will this relative "link" be automatically taken care of?

File: content/profiles/shyam-upadhyaya.md [hunk 2]
```diff
@@ -6,4 +6,4 @@ organization: United Nations Industrial Development Organization (UNIDO)
 avatar: ../assets/profiles/shyam-upadhyaya.jpg
 ---
 
-Shyam Upadhyaya is the Chief Statistician for [UNIDO](https://www.unido.org/). He started his career with the [Central Bureau of Statistics of Nepal](http://www.cbs.gov.np/), and has worked as an international consultant on capacity building projects for national statistical offices. He was actively involved in the work of the [IAEG-SDG](https://unstats.un.org/sdgs/iaeg-sdgs/) in preparing the [SDG indicator framework](https://unstats.un.org/sdgs/).
+Shyam Upadhyaya is the Chief Statistician for [UNIDO](https://www.unido.org/). He started his career with the [Central Bureau of Statistics of Nepal](http://www.cbs.gov.np/), and has worked as an international consultant on capacity building projects for National Statistical Offices. He was actively involved in the work of the [Inter-agency and Expert Group on Sustainable Development Goal Indicators](https://unstats.un.org/sdgs/iaeg-sdgs/) in preparing the [Sustainable Development Goal indicators framework](https://unstats.un.org/sdgs/).
```
Comment (gonzalezmorales) [inline content/profiles/shyam-upadhyaya.md]: [Inter-agency and Expert Group on Sustainable Development Goal Indicators (IAEG-SDGs)](https://unstats.un.org/sdgs/iaeg-sdgs/)

-- This acronym is important in the SDG community.  It's part of the GA 70/1 resolution.

File: content/posts/public-private-partnerships-in-statistical-capacity-development.md (hunk 1 above)
Comment (454de6e) [reply content/posts/public-private-partnerships-in-statistical-capacity-development.md]: I would rather put in a broken link than no link. So I am making up URLs. The idea is then that we detect broken links later via Google Webmaster tools when Google indexes the site. If there is no link we won't go back. Alternatively we would need to open an issue for each such link and fix it later. Maybe better to open issues and tag them with `missing link` or something? What do you think, @gonzalezmorales ?

Comment (gonzalezmorales) [reply content/posts/public-private-partnerships-in-statistical-capacity-development.md]: Do I understand correctly that the behavior of such broken links is "nothing happens" (akin to a placeholder link of the form <a href="#">link text</a>) ?  Keeping a broken link as you suggest is ok as long as users do not get directed to a 404.

Comment (454de6e) [reply content/posts/public-private-partnerships-in-statistical-capacity-development.md]: Yes they will end on a 404. Then we need to create no link and open issue for each link to be added later.

**Pull request merged**